src/scss/_variables.scss

```diff
@@ -49,37 +49,94 @@ $o-typography-font-scale: (
    10: (84, 84),
 );
 
+$o-typography-font-scale-by-font: (
+	'MetricWeb': (
+		-2: (12, 16),
+		-1: (14, 16),
+		0: (16, 20),
+		1: (18, 20),
+		2: (20, 24),
+		3: (24, 28),
+		4: (28, 32),
+		5: (32, 32),
+		6: (40, 40),
+		7: (48, 48),
+		8: (56, 56),
+		9: (72, 72),
+		10: (84, 84)
+	),
+	'FinancierDisplayWeb': (
+		-2: (12, 16),
+		-1: (14, 16),
+		0: (16, 20),
+		1: (18, 20),
+		2: (20, 24),
+		3: (24, 28),
+		4: (28, 32),
+		5: (32, 32),
+		6: (40, 40),
+		7: (48, 48),
+		8: (56, 56),
+		9: (72, 72),
+		10: (84, 84)
+	),
+	'Georgia': (
+		-2: (12, 16),
+		-1: (14, 16),
+		0: (16, 20),
+		1: (18, 20),
+		2: (20, 24),
+		3: (24, 28),
+		4: (28, 32),
+		5: (32, 32),
+		6: (40, 40),
+		7: (48, 48),
+		8: (56, 56),
+		9: (72, 72),
+		10: (84, 84)
+	)
+);
+
+/// @todo move into o-fonts, add mixin to o-fonts to add new font
+$o-fonts-families: map-merge($o-fonts-families, ('Georgia': (
+	font-family: 'Georgia, serif',
+	variants: (
+		(weight: regular, style: normal),
+		(weight: regular, style: italic),
+		(weight: semibold, style: normal),
+		(weight: semibold, style: italic),
+		(weight: bold, style: normal),
+		(weight: bold, style: italic)
+	)
+)));
+
 /// Configuration for fallback fonts when loading
 /// fonts progressively
 /// @type Map
 $_o-typography-progressive-font-fallbacks: (
-	sans: (
-		font: $o-typography-sans,
+	(
+		label: 'sans',
+		family: $o-typography-sans,
 		fallback: sans-serif,
 		fallback-scale: 0.87
 	),
-	sansBold: (
-		font: $o-typography-sans,
+	(
+		label: 'sansBold',
+		family: $o-typography-sans,
+		weight: ('bold', 'semibold'),
 		fallback: sans-serif,
 		fallback-scale: 0.83
 	),
-	sansItalic: (
```

Comment:
Don't we need to keep this until the major?

Reply:
It's redundant, not used/implemented, and the variable is private.